client/src/features/projectsV2/fields/EditProjectMemberModal.tsx

```diff
@@ -109,7 +107,14 @@ function EditProjectMemberAccessForm({
           <div
             className={cx("align-items-baseline", "d-flex", "flex-row", "mb-3")}
           >
-            <Label>{member.email ?? member.id}</Label>
+            <Label>
+              {userName}{" "}
+              {member.namespace ? (
+                <span className="fst-italic">@{member.namespace}</span>
+              ) : (
+                ""
+              )}
```

Comment:
Building on the suggested change in `roleUtils`: 
```suggestion
              <ProjectMemberDisplay member={member} />
```